src/toast/ops/stokes_weights/stokes_weights.py

```diff
@@ -149,6 +152,10 @@ def _exec(self, data, detectors=None, use_accel=None, **kwargs):
         if self.detector_pointing is None:
             raise RuntimeError("The detector_pointing trait must be set")
 
+        if (self.mode == "IQU") and self.hwp_angle is not None:
```

Comment:
There could be other pointing modes that require `gamma`. Namely, any polarized mode, including `QU` and `IQUSS`.  How about checking  `if "QU" in self.mode` instead?

Reply:
Changed.